### PR TITLE
Reworks Power Puzzle

### PR DESCRIPTION
**Pull request by goober3**
Reworks Power Puzzle to (hopefully?) fit modern Testing standards.

## Holy Shit A New Ruin

d

image

e

old

Loots: -Hand Teleporter
-Refined Bluespace Crystal
-Bluespace Anomaly Core
-Commander + x2 Surplus Ammo Boxes / magazines
-250 credits
-Insulated Gloves
-Cell Charger Board
-SUPERPACMAN generator
-4 ½ EVA suits
-Engi-Vend
-Big Bess + exosuit drill, hydraulic clamp, ion thruster package (hidden)
-Exotic........Corgi............

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

It's A Ruin, Holy Shit!

## Changelog

:cl:
tweak: Reworked PowerPuzzle, a cargo facility hidden in the outer reaches of space.
/:cl:

### CODE REVIEW

File: code/game/area/areas/ruins/space.dm [hunk 1]
```diff
@@ -376,40 +376,30 @@
 	name = "Mac Space Restaurant"
 	icon_state = "yellow"
 
-//POWER PUZLE
+//NUCLEAR DUMP -- this ruin uses an area from power puzzle, for whatever reason. added new areas, for now.
```
Comment (Erikafox) [inline code/game/area/areas/ruins/space.dm:379]: good catch

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
This room needs a lil bit of decalling
StrongDMM_RmceMpJYgB
Ask a lore fellow about the hand teleporter. I think it's fine but they may think otherwise.

**Comment by goober3** [comment]
> image
This room needs a lil bit of decalling

image


>Ask a lore fellow about the hand teleporter. I think it's fine but they may think otherwise.

I spoke to Valorium about this, the hand tele should be Fine!

**Review by ZephyrTFA — APPROVED** [review]

**Comment by Apogee-dev** [comment]
You have a few railings in the middle room blocking part of the main pathway that seem really out of place. Also, I'd suggest using Nanotrasen engineer corpses, the ones you have right now are dressed in indie outfits. You might have to make a new object for those if it's not already available.

**Comment by goober3** [comment]
> You have a few railings in the middle room blocking part of the main pathway that seem really out of place. Also, I'd suggest using Nanotrasen engineer corpses, the ones you have right now are dressed in indie outfits. You might have to make a new object for those if it's not already available.

oops. did not mean for those to be there. Fixing Now.

**Review by Erikafox — APPROVED** [review]

**Pull request merged**